README.md

```diff
@@ -140,6 +140,20 @@ useful with a minimal barrier to adoption.
 
 ## Advanced Usage
 
+### Installing specific versions of uv
```

Comment:
Can we move this lower down? It seems fringe compared to other topics here.

Reply:
I don't think it's fringe, I feel like pinning the version is good practice and pretty normal?

Reply:
Do you want it under the entire advanced usage section?

Reply:
Moved down